lib/bundler/cli/plugin.rb

```diff
@@ -5,14 +5,16 @@ module Bundler
   class CLI::Plugin < Thor
     desc "install PLUGINS", "Install the plugin from the source"
     long_desc <<-D
-      Install plugins either from the rubygems source provided (with --source option) or from a git source provided with (--git option). If no sources are provided, it uses Gem.sources
+      Install plugins either from the rubygems source provided (with --source option) or from a remote/git source provided (with respectively --git and --local_git options). If no sources are provided, it uses Gem.sources
```

Comment:
Just rereading this and I wonder if we can make it clearer. What do you think of:

```
Install plugins either from the rubygems source provided (with --source option) or from a git source provided with --git (for remote repos) or --local_git (for local repos.) If no sources are provided, it uses Gem.sources
```

Too long?

Reply:
:+1: for me.